mlocal/frags/Makefile.stub

```diff
@@ -27,13 +27,26 @@ check: $(go_OBJ)
 dist:
 	$(V)(cd $(SOURCEDIR) && $(SOURCEDIR)/scripts/make-dist.sh)
 
+.PHONY: unit-test
+unit-test:
+	@echo " TEST sudo go test [unit]"
+	@cd $(SOURCEDIR) && sudo -E `which go` test -count=1 -timeout=20m -tags "$(GO_TAGS)" -failfast -cover -race \
+		`go list ./... | grep -v "cmd/singularity\|pkg/network"`
+	@echo "       PASS"
+
+
+.PHONY: integration-test
+integration-test:
+	@echo " TEST sudo go test [integration]"
+	@cd $(SOURCEDIR) && sudo -E `which go` test -count=1 -timeout=20m -tags "$(GO_TAGS)" -failfast -cover -race \
+		./cmd/singularity ./pkg/network
+	@echo "       PASS"
+
+
 .PHONY: test
 test:
-	@echo " TEST sudo go test"
-	@# avoid travis 10 minutes of silence timeout
-	$(V)M=0; eval 'while [ $$M -le 20 ]; do sleep 60; M=`expr $$M + 1`; echo "Still testing ($$M) ..."; done &' ; \
```

Comment:
Think we need to keep this for now, while we have Travis in the mix (even if it's not a requirement to merge.) Otherwise he is unhappy:

```
 TEST sudo go test [all]
?   	github.com/sylabs/singularity/cmd/bash_completion	[no test files]
?   	github.com/sylabs/singularity/cmd/docs	[no test files]
ok  	github.com/sylabs/singularity/cmd/internal/cli	1.090s	coverage: 25.6% of statements
?   	github.com/sylabs/singularity/cmd/plugin_manifestgen	[no test files]
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```